### PR TITLE
Deploying Unlock to Kovan network

### PR DESCRIPTION
**Pull request by nfurfaro**
# Description
Unlock.sol has been deployed to kovan. These are the changes that were needed in `truffle-config.js` , and the resulting zos network file.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [x] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: smart-contracts/truffle-config.js [hunk 1]
```diff
@@ -98,7 +124,7 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: '0.5.12',
+      version: '0.5.9',
```
Comment (NickCuso) [inline smart-contracts/truffle-config.js]: need to revert this change

Comment (nfurfaro) [reply smart-contracts/truffle-config.js]: yup, will do.

File: smart-contracts/truffle-config.js [hunk 2]
```diff
@@ -89,6 +109,12 @@ module.exports = {
       gas: 5000000,
       gasPrice: 5000000000, // 5GWEI
     },
+    kovan: {
+      provider: kovanProvider,
+      network_id: '42', // Network Id for Rinkeby
```
Comment (NickCuso) [inline smart-contracts/truffle-config.js]: Comment error - rinkeby vs kovan
